frontend/src/services/agent.js

```diff
@@ -73,6 +73,10 @@ const Psychologist = {
   getProfile: id => client.get(`/psychologist/${id || store.userStore.user.dossierNumber}`),
   suspend: (reason, date) => client
     .post(`/psychologist/${store.userStore.user.dossierNumber}/suspend`, { reason, date }),
+  inactive: (reason, token) => client
```

Comment:
Why not reuse `suspend` endpoint?

Reply:
because suspend does not change the confirm boolean in the db